### ISSUE TITLE
Webster 1828

### ISSUE THREAD

**Issue by luke-2136** [AndBible/and-bible#2021]
I know you do not make the modules but this one in particular - Webster 1828 - says it is a 'AndBible Extra' so maybe you did this one; anyway the issue is that it shows an html link on some words but apparently whatever it wants to link to is missing. 

Here is a random word for example - displays this: 

Abarticulation, n. [See <see target="x-self">tid=3676</see>.]

*only if this module is done by AndBible of course; thank you [on Android 10]

**Comment by luke-2136** [comment]
*tried to edit above comment but the three dots only bring up a swirly..

anyway looks like above the html example was displayed as an actual link

{it actually looks like this [without the opening and closing < >. ]:    see target="x-self">tid=3676</see

**Comment by tuomas2** [comment]
"And Bible Extra" is actually copy of Xiphos repository (which was served only via ftp or http (not https) when I copied this, thus could not be used directly by And Bible).

**Comment by agrogers** [comment]
Is it still using the copy or does Xiphos support secure connections now?

**Comment by tuomas2** [comment]
We are using AB Extra still. Xiphos supports secure connections (as far as I know) but Xiphos repository is not available via https.

**Comment by tuomas2** [comment]
I guess this is out of scope for And Bible project, but we can release improved version if someone figures out how to proceed from here.

**Comment by Noitarud** [comment]
How do you edit the resource? The file is not in readable characters…

**Comment by tuomas2** [comment]
You need to have source file (often OSIS xml), make edits there, and re-compile the module (in case of OSIS source, with osis2mod). Sometimes xml files are made public (like Crosswire does for some modules: https://gitlab.com/crosswire-bible-society) and sometimes not. You could also try to contact original module creator or Xiphos repository maintainer.

**Comment by Noitarud** [comment]
I began to look into it, I found the original source site, but there have been some detrimental updates in the present version (noticed zoöl to zool, abbreviation in "velvet") [available here](http://www.ibiblio.org/webster/), sent email.

The links are rudimentary, just \<er>Love\</er> for the inside links (treats spn as links too). 

Then there's [the one on Crosswire](https://gitlab.com/crosswire-bible-society/webster1913), dated internally 2008. initial thought is I noticed the pronounciation characters are visible. Again there's nothing sophisticated in the links.

so it not likely a problem in the source document, but in osis2mod (at the time), or else an intermediate preparation step broke it. As for the pronounciation, again I suspect osis2mod not handling Unicode (I doubt a problem with AndBible). So will have to try a recompile see if they have fixed it.

If only the XML could be directly used :P should only need to compile an index file (addresses) for it.
